Source/WebCore/svg/SVGLengthValue.cpp

```diff
@@ -350,35 +430,59 @@ ExceptionOr<void> SVGLengthValue::setValueAsString(StringView string)
     if (string.isEmpty())
         return { };
 
-    return readCharactersForParsing(string, [&](auto buffer) -> ExceptionOr<void> {
-        auto convertedNumber = parseNumber(buffer, SuffixSkippingPolicy::DontSkip);
-        if (!convertedNumber)
+    // FIXME: SVG2 allows trailing spaces?
```

Comment:
Can you write this in a way that’s clearer? What is the question? How would we change the code based on the answer? Why is it better to have a `FIXME` instead of a test case?